### PR TITLE
Integrate with AutoComplete component

### PR DESCRIPTION
**Pull request by GlistenSTAR**

### CODE REVIEW

**Comment by blokhin** [comment]
@GlistenSTAR please, let's return back to the lower-case element group names 🙏

**Comment by GlistenSTAR** [comment]
> @GlistenSTAR please, let's return back to the lower-case element group names 🙏

Okay 👌👌👌

File: src/www/docs/components/periodictable/+page.md [hunk 1]
```diff
@@ -9,10 +9,20 @@ config: {}
...
 
 <script>
-	import { PeriodicTable } from '$lib'
-	import {Button, Modal} from '$lib'
-	let selected = [], clear;
-	let open = false;
+	import { PeriodicTable, Button, Modal, Autocomplete } from '$lib'
+
+	let selected = [], open = false, clear;
+    let predefined = ["H", "He", "Li", "Be", "B", "C", "N", "O", "F", "Ne", "Na", "Mg", "Al", "Si", "P", "S", "Cl", "Ar", "K", "Ca", "Sc", "Ti", "V", "Cr", "Mn", "Fe",     "Co", "Ni", "Cu", "Zn", "Ga", "Ge", "As", "Se", "Br", "Kr", "Rb", "Sr", "Y", "Zr", "Nb", "Mo", "Tc", "Ru", "Rh", "Pd", "Ag", "Cd", "In", "Sn", "Sb", "Te", "I", "Xe", "Cs", "Ba", "La", "Ce", "Pr", "Nd", "Pm", "Sm", "Eu", "Gd", "Tb", "Dy", "Ho", "Er", "Tm", "Yb", "Lu", "Hf", "Ta", "W", "Re", "Os", "Ir", "Pt", "Au", "Hg", "Tl", "Pb", "Bi", "Po", "At", "Rn", "Fr", "Ra", "Ac", "Th", "Pa", "U", "Np", "Pu", "Am", "Cm", "Bk", "Cf", "Es", "Fm", "Md", "No","period 1", "period 2", "period 3", "period 4", "period 5", "period 6", "period 7", "lanthanide", "actinide", "alkali", "alkaline", "group 3", "group 4", "group 5", "group 6", "group 7", "group 8", "group 9", "group 10", "group 11", "group 12", "triels", "tetrels", "pnictogen", "chalcogen", "halogen", "noble gas"]
```
Comment (blokhin) [inline src/www/docs/components/periodictable/+page.md:15]: @GlistenSTAR please, could we avoid duplicating the names? Importing them from a component would be much better.

Comment (GlistenSTAR) [reply src/www/docs/components/periodictable/+page.md:15]: `predefined` is variable for autocomplete.

**Review by blokhin — APPROVED** [review]

**Review by blokhin — APPROVED** [review]

**Pull request merged**